configurations/configurations_test.go

```diff
@@ -6,7 +6,7 @@ import (
 	"testing"
 )
 
-var expectedGenericOutput = `FROM golang:latest
+var expectedCodeRunnerOutput = `FROM golang:latest
```

Comment:
This looks like the expected content of the Dockerfile created correct? If yes, then may be name the variable `expectedCodeRunnerDockerfileContents`.

Reply:
Okay